lib/active_job/queue_adapters/sidekiq_adapter.rb

```diff
@@ -1,19 +1,5 @@
 # frozen_string_literal: true
 
-# Define a new class whose name is shorter than from Active Job.
```

Comment:
To make the PR diff shorter, I would left this class as is and just change a few lines down in the class `"class" => Sidekiq::ActiveJob::Wrapper,` to use the old name (note that this change is needed in any case, because we still enqueue the new class name).

Reply:
Not sure I follow, but if we want to alias the constant it has to be defined first, right?

Reply:
I mean the code change for this PR will be only something like 
```diff
diff --git a/lib/active_job/queue_adapters/sidekiq_adapter.rb b/lib/active_job/queue_adapters/sidekiq_adapter.rb
index 2bfa91d6..ad14cb9e 100644
--- a/lib/active_job/queue_adapters/sidekiq_adapter.rb
+++ b/lib/active_job/queue_adapters/sidekiq_adapter.rb
@@ -34,7 +34,7 @@ module ActiveJob

       # @api private
       def enqueue(job)
-        job.provider_job_id = Sidekiq::ActiveJob::Wrapper.set(
+        job.provider_job_id = JobWrapper.set(
           wrapped: job.class,
           queue: job.queue_name
         ).perform_async(job.serialize)
@@ -42,7 +42,7 @@ module ActiveJob

       # @api private
       def enqueue_at(job, timestamp)
-        job.provider_job_id = Sidekiq::ActiveJob::Wrapper.set(
+        job.provider_job_id = JobWrapper.set(
           wrapped: job.class,
           queue: job.queue_name
         ).perform_at(timestamp, job.serialize)
@@ -57,7 +57,7 @@ module ActiveJob

             if immediate_jobs.any?
               jids = Sidekiq::Client.push_bulk(
-                "class" => Sidekiq::ActiveJob::Wrapper,
+                "class" => JobWrapper,
                 "wrapped" => job_class,
                 "queue" => queue,
                 "args" => immediate_jobs.map { |job| [job.serialize] }
@@ -67,7 +67,7 @@ module ActiveJob

             if scheduled_jobs.any?
               jids = Sidekiq::Client.push_bulk(
-                "class" => Sidekiq::ActiveJob::Wrapper,
+                "class" => JobWrapper,
                 "wrapped" => job_class,
                 "queue" => queue,
                 "args" => scheduled_jobs.map { |job| [job.serialize] },
```

And then in the sidekiq 8.0 we will remove that `JobWrapper = Sidekiq::ActiveJob::Wrapper` alias and switch back to enqueuing shorter job names via `"class" => Sidekiq::ActiveJob::Wrapper`.

Reply:
Oh yeah, good catch. Thought you meant the diff of moving this module to the bottom. Thanks for the review!